packages/api-page-builder-aco/src/utils/getLexicalContentText.ts

```diff
@@ -0,0 +1,32 @@
+export const getLexicalContentText = (value: string): string => {
+    if (!isValidJSON(value)) {
+        return value;
+    }
+
+    return traverse(JSON.parse(value), "text");
```

Comment:
I would recommend returning the parsed value from `isValidJson` to avoid double parsing. Better name the function `parseJSON`, and if it throws an error, return the original value. If it returns successfully, you have your object for traversal.

Reply:
I just simplified it as you suggested, handling the error `catch` within `getLexicalContentText` function.